db/schema.rb

```diff
@@ -10,32 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170115230058) do
+ActiveRecord::Schema.define(version: 20170123173604) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
   create_table "activities", force: :cascade do |t|
     t.integer  "team_id"
-    t.string   "kind",         limit: 255
-    t.string   "guid",         limit: 255
-    t.string   "author",       limit: 255
-    t.string   "title",        limit: 255
+    t.string   "kind"
```

Comment:
I've yet to get behind how Rails generates the schema.rb. It would seem the schema dump depends heavily on the actual pgsql patchlevel used when the migrations were run. :woman_shrugging: